src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,20 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery firstBall = new Lottery();
+        firstBall.getRandomBall().getColor();
+        firstBall.getRandomBall().getNumber();
+
+        Lottery secondBall = new Lottery();
+        secondBall.getRandomBall().getColor();
+        secondBall.getRandomBall().getNumber();
+
+        Lottery thirdBall = new Lottery();
+        thirdBall.getRandomBall().getColor();
+        thirdBall.getRandomBall().getNumber();
+
+        System.out.println(firstBall);
+        System.out.println(secondBall);
+        System.out.println(thirdBall);
```

Comment:
use for loop + you don't need to call getColor and getNumber methods only System.out.println of lottery.getRandomBall()